docs/api/Query.md

```diff
@@ -112,7 +114,9 @@ Cat.query("name").eq("Will").using("name-index"); // Run the query on the `name-
 
 ## query.all([delay[, max]])
 
-Unlike most other query functions that directly change the DynamoDB query request, this function is purely internal and unique to Dynamoose. This function sends continuous query requests upon receiving the response so long as the `lastKey` property exists on the response. This can be useful if you wish to get all the items from the table and don't want to worry about checking the `lastKey` property and sending a new query request yourself.
+If a query result is more than 1 MB (before filtering!), DynamoDB paginates the results so you would have to send multiple requests. Please see the [AWS DynamoDB documentation](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Query.Pagination.html) for further informations.
```

Comment:
```suggestion
Due to the fact that [DynamoDB paginates query results that are more than 1 MB](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Query.Pagination.html) (before filtering!), Dynamoose provides the `query.all` method to automatically make subsequent requests to DynamoDB.
```

Reply:
Also not sure I'm a fan of having AWS links in the documentation. It's one more thing to maintain in case of 404 errors or something.

It should be assumed that if you are gonna use Dynamoose that you need to research DynamoDB. I don't think Dynamoose needs to be responsible for doing that.

I'm totally willing to reconsider that in the future maybe. But this is still a small project in terms of number of maintainers. Adding a link to AWS docs and trying to education people about DynamoDB I don't think should be part of Dynamoose documentation at this point when I don't have enough support in maintaining the project.

Also, the 1 MB limit, that could change at any point. Which again, is another point of having to maintain and update things. It just increases the chances of things getting out of date extremely quickly.

So far I've tried to give good advice to anyone asking for it. But going in depth to DynamoDB in the docs is a whole different game.

Reply:
I changed to:

> If a query result is more than the limit of your DynamoDB table (before filtering!), DynamoDB paginates the results so you would have to send multiple requests.

What do you think?